<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Niemas-MacBook-Pro.local" ignored="0" name="API" tests="3" failures="2" timestamp="19 Jan 2020 21:17:59 GMT" time="3.576" errors="0">
  <testcase name="GetPostsResponse" time="1.776" classname="Service.apiTest"/>
  <testcase name="PostNewUserResponse" time="0.838" classname="Service.apiTest">
    <failure type="java.lang.AssertionError" message="OPERATION_SUCCESS expected [null] but found [Correct Success code was returned]">
      <![CDATA[java.lang.AssertionError: OPERATION_SUCCESS expected [null] but found [Correct Success code was returned]
at Service.apiTest.PostNewUserResponse(apiTest.java:62)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- PostNewUserResponse -->
  <testcase name="PutUserInfo" time="0.201" classname="Service.apiTest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at Service.apiTest.PutUserInfo(apiTest.java:87)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- PutUserInfo -->
</testsuite> <!-- API -->
